lib/tokens/token.js

```diff
@@ -40,28 +40,34 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
     this.algorithm = 'sha256'
     this.uid = details.uid || null
     this.lifetime = details.lifetime || Infinity
-    this.createdAt = optionallyOverrideCreatedAt(details.createdAt)
+    this.createdAt = details.createdAt || 0
```

Comment:
Probably unnecessary; `details.createdAt` will either have been set to the current time or overridden value in `createNewToken`, or to can existing value read from the db in `createTokenFromHexData`.  Perhaps it would be more correct to *throw* here if `createdAt` is undefined, rather than defaulting to zero?